src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationService.kt

```diff
@@ -68,7 +67,7 @@ class TierCalculationService(
     )
 
     return tierCalculationRepository.save(calculation).also {
-      log.info("Calculated tier for $crn")
+      log.info("Calculated tier for $crn ${calculation.data}")
```

Comment:
I'm not sure we can log this at info because it has the CRN and the tier in the same message. 
Anything logged at Info will end up in Application Insights, but Debug should only be output on dev machines

Reply:
Taking this to its logical conclusion, we shouldn't log the calculated tier anywhere at Info then really, as it's going to be easily tied back to the CRN in ApplicationInsights based on the sequence of events